chromium_src/base/logging/rust_log_integration.cc

```diff
@@ -19,8 +19,10 @@ BASE_EXPORT void print_rust_log(const char* msg,
                                 bool verbose) {
   switch (severity) {
     // Trace and debug logs are set as `LOGGING_INFO`. Trace is also set as
-    // versbose, so we make a higher level verbosity.
+    // verbose, so we make a higher level verbosity. We also map `LOGGING_WARN`
```

Comment:
why did we need to override this originally before this change?

Reply:
It was spamming vlog on CI when building with DCHECKs always on, and causing CI to fail.